packages/angular_devkit/build_angular/src/extract-i18n/schema.json

```diff
@@ -49,7 +49,8 @@
     },
     "ivy": {
       "type": "boolean",
-      "description": "Use Ivy compiler to extract translations. The default for Ivy applications."
+      "description": "Use Ivy compiler to extract translations. The default for Ivy applications.",
+      "x-deprecated": "Option is always enabled. Angular version 12 and higher no longer support View Engine compilation for applications."
```

Comment:
Should we just remove anice it’s a noop and the package is technically experimental

Reply:
Ok